sdk/deviceregistry/arm-deviceregistry/package.json

```diff
@@ -104,9 +104,9 @@
     "test:browser": "npm run clean && npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
     "test:node": "npm run clean && dev-tool run build-package && npm run unit-test:node && npm run integration-test:node",
     "unit-test": "npm run unit-test:node && npm run unit-test:browser",
-    "unit-test:browser": "npm run build:test && dev-tool run test:vitest --browser",
+    "unit-test:browser": "echo skipped",
```

Comment:
Did you mean to skip these?

Reply:
Yes, by default, all ARM packages skip browser testing.  This one was an outlier from an earlier migration.